src/jit.c

```diff
@@ -1349,7 +1350,7 @@ void parse_branch_history_table(struct jit_state *state, rv_insn_t *ir)
         if (bt->times[max_idx] < bt->times[i])
             max_idx = i;
     }
-    if (bt->PC[max_idx]) {
+    if (bt->PC[max_idx] && bt->times[max_idx] >= 256) {
```

Comment:
Why 256 here?

Reply:
As mentioned in the first commit of this PR, I set 256 as the threshold because we recorded 16 indirect jumps, and
the threshold for launching T1C is 4096. 256 == 4096 / 16. I think the indirect jump with the highest frequency should exceed this threshold.

Reply:
Avoid hardcoded value. Use certain identifier instead.